auth.js

```diff
@@ -10,22 +10,24 @@ require('dotenv').config();
 
 /*
 * My understanding of passport API:
+* Frontend: User sees log in button. They click log in button.
+* Frontend: Browser sends a request to http://localhost:5000/auth/login from the <a> tag.
+* Backend: Request hits express route /auth/login, where passport api logins user with the auto0 strategy.
 * 
 */
 
 router.get(
-  '/login',
+  '/login', 
```

Comment:
get rid of trailing space

```suggestion
  '/login',
```